cpp/tests/strings/chars_types_tests.cpp

```diff
@@ -14,10 +14,14 @@
  * limitations under the License.
  */
 
+#include <cudf/strings/convert/is_valid_element.hpp>
```

Comment:
I would suggest moving these to a new test source file `/cpp/tests/strings/valid_element.cpp`